src/events/events.client.ts

```diff
@@ -13,36 +13,28 @@ export class EventsClient implements CommandClient {
 
   constructor(
     fallback?: Event[]
-    ) {
-      this.events = fallback || events;
-
+  ) {
+    this.events = fallback || events;
+    this.commandNode = new CommandNodeBuilder()
+      .withCommand(/events/, msg => {
+        const upcomingEvents = this.events.map(e => `[${e.name}](${e.link})`).join(', ');
+        return this.mb.message(`Events for this year include ${upcomingEvents}`);
+      })
+      .withName('events')
+      .withChild(b => b.withCommand(/\w/, this.command))
```

Comment:
with child is just the process of adding a child node to the current node
so with a structure like 
parent -> child
where parent is something like `events` and child is something like `events :query`
you say parent.withChild(somechild)